src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleObject.cs

```diff
@@ -52,9 +52,12 @@ private static void HandleStartObject(JsonSerializerOptions options, ref Utf8Jso
             }
         }
 
-        private static void HandleEndObject(JsonSerializerOptions options, ref Utf8JsonReader reader, ref ReadStack state)
+        private static void HandleEndObject(ref Utf8JsonReader reader, ref ReadStack state)
```

Comment:
Why do we pass in the reader to `HandleEndObject` but `JsonSerializerOptions` to `HandleStartObject`?

Reply:
It's just based on what is used within the method